### PR TITLE
fix(adapter): make dbt-fal~1.5 work with redshift

### PR DESCRIPTION
**Pull request by mederka**

### CODE REVIEW

File: projects/adapter/pyproject.toml [hunk 1]
```diff
@@ -54,7 +55,7 @@ pytest = "^5.2"
 postgres = []
 snowflake = ["snowflake-connector-python"]
 bigquery = ["google-cloud-bigquery"]
-redshift = ["sqlalchemy-redshift"]
+redshift = ["sqlalchemy-redshift", "awswrangler"]
```
Comment (chamini2) [inline projects/adapter/pyproject.toml:58]: do we still need `sqlalchemy-redshift`?

**Review by chamini2 — COMMENTED** [review]
Looks pretty good!

File: projects/adapter/pyproject.toml [hunk 2]
```diff
@@ -34,6 +34,7 @@ google-cloud-bigquery = { version = "~3.5.0", extras = ["pandas"], optional = tr
 
 ## redshift
 sqlalchemy-redshift = { version = "^0.8.9", optional = true }
+awswrangler = { version = ">=3.0.0", extras = ["redshift"], optional = true}
```
Comment (chamini2) [inline projects/adapter/pyproject.toml:37]: also curious if you considered the one used by dbt-redshift?

https://github.com/dbt-labs/dbt-redshift/blob/b644963a6bf7d3e64bd71e6ee97a205900089db1/setup.py#L87-L88

File: projects/adapter/pyproject.toml (hunk 1 above)
Comment (mederka) [reply projects/adapter/pyproject.toml]: Good catch! I don't think so.

File: projects/adapter/pyproject.toml (hunk 2 above)
Comment (mederka) [reply projects/adapter/pyproject.toml]: Yeah, thank you for suggesting, @chamini2! awswrangler just seemed tidier. They also have pretty good Athena support, so I was thinking that maybe we use it for Athena as well.

Comment (mederka) [reply projects/adapter/pyproject.toml]: https://github.com/aws/aws-sdk-pandas

Comment (chamini2) [reply projects/adapter/pyproject.toml]: Very interesting! Let's try that!

**Review by chamini2 — APPROVED** [review]
Awesome!

**Pull request merged**